manifests/mlops/core-modules.yaml

```diff
@@ -0,0 +1,74 @@
+name: eks
+path: modules/core/eks/
+parameters:
+  - name: vpc-id
+    valueFrom:
+      moduleMetadata:
+        group: optionals
+        name: networking
+        key: VpcId
+  - name: private-subnet-ids
+    valueFrom:
+      moduleMetadata:
+        group: optionals
+        name: networking
+        key: PrivateSubnetIds
+  - name: eks-admin-role-name
+    value: Admin
+  - name: eks-compute
+    value:
+      eks_nodegroup_config:
+        - eks_ng_name: ng1
+          eks_node_quantity: 3
+          eks_node_max_quantity: 6
+          eks_node_min_quantity: 2
+          eks_node_disk_size: 50
+          eks_node_instance_types:
+            - "m5.large"
+        - eks_ng_name: ng2
```

Comment:
I think single node-group will be enough for the sample deployment.
We can comment out the other two node-groups.

Reply:
We have used these 2 node groups as examples as the standard.  I added the 3rd group to demonstrate ML compute.  I disagree with your statement and prefer to leave it as-is.

Reply:
Ok, let's leave it as it is.